### ISSUE TITLE
Build script adds 'v' prescript to version number

### ISSUE THREAD

**Issue by halcwb** [TheAngryByrd/MiniScaffold#213]
**Describe the bug**
When publishing a version, the build script adds a `v` prefix to the version number.

**To Reproduce**
1. Run a publish to nuget target using a version number like 1.0.1

**Expected behavior**
Expected referencing the version as `1.0.1` not `v.1.0.1`

**Example result**

Release Notes
## [1.0.1] - 2020-05-09

[1.0.1]: https://github.com/halcwb/GenBuild/compare/v0.2.1-beta...v1.0.1

### Changed
* Using MiniScaffold as template
* Moving to dotnet and local cli support
* Fixing various problems with updating using MiniScaffold

Note the `compare/v0.1.1-beta..v1.0.1`, should be `compare/0.1.1-beta1.0.1`.

**Comment by TheAngryByrd** [comment]
This is working as expected.  If you don't want to have that added you can remove the prefix in your script.

**Comment by halcwb** [comment]
@TheAngryByrd, I am sorry, but where do I remove the prefix in the build script?

**Comment by TheAngryByrd** [comment]
You're right I should have specified where: https://github.com/TheAngryByrd/MiniScaffold/blob/master/Content/Library/build.fsx#L82

**Comment by halcwb** [comment]
Thanks again!

**Comment by TheAngryByrd** [comment]
I thought I had this discussion about why we went that way in https://github.com/TheAngryByrd/MiniScaffold/pull/175 but it doesn't seem to exist. We must have chatted on slack or something about it.

@rmunn can you discuss the rationale for prefixing with `v` ?

**Comment by rmunn** [comment]
Prefixing the version number with `v` appears to be the *de facto* standard. These are the reasons why I used it:

- First, most projects that I've seen use that convention, tagging version 1.2.3 with the Git tag `v1.2.3`.
- The first Stack Overflow question I found while Googling just now [was overwhelmingly in favor of the `v` prefix](https://stackoverflow.com/questions/21639437/git-flow-release-branches-and-tags-with-or-without-v-prefix).
- No possibility of confusion with a Git commit hash, whereas a version number without a leading `v`, if you use dates for version numbers, can look like a hash prefix (`20200509` would be a valid Git commit hash, but `v20200509` would not since commit hashes are hex digits).
- GitHub actually recommends using the leading `v`: if you go to https://github.com/{username}/{repo}/releases/new and look in the right-hand column, you'll see the text "**Tagging suggestions**: It’s common practice to prefix your version names with the letter v. Some good tag names might be v1.0 or v2.3.4."

So I decided to let the build script default to the *de facto* standard convention. If you want a different convention, as @TheAngryByrd pointed out, you just need to change the one-line `tagFromVersionNumber` function.
